vitest.config.ts

```diff
@@ -0,0 +1,10 @@
+// vite.config.ts
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+	test: {
+		coverage: {
+			// provider: "istanbul", // or 'c8'
+		},
+	},
+});
```

Comment:
Todo: I might end up removing this file depending on which coverage provider works. And then I'll remove the unused coverage provider from `package.json`.